helm/Chart.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
   - name: opentelemetry-collector
     repository: https://open-telemetry.github.io/opentelemetry-helm-charts
     condition: opentelemetry-collector.enabled
-    version: 0.55.2
+    version: 0.100.0
```

Comment:
In our Docker Compose setup we are already on version `docker.io/otel/opentelemetry-collector:0.116.1` so we are quite out of sync already.. Is Open Telemetry and this collector even already part of the overall PodiumD Helm Chart @bas-info-nl @infojohn ? Probably we should upgrade this now anyway.

Reply:
Not part of the PodiumD helm chart, but might (should) be in the future. For us we should update this either way 👍